backend/docker/docker-compose-full.yml

```diff
@@ -59,3 +65,23 @@ services:
             interval: 5s
             timeout: 2s
             retries: 20
+
+    minio:
+        image: minio/minio
+        ports:
+            - "9000:9000"
+            - "9001:9001"
+        volumes:
+            - minio_data:/data
+        environment:
+            MINIO_ROOT_USER: ${MINIO_ACCESS_KEY}
+            MINIO_ROOT_PASSWORD: ${MINIO_SECRET_KEY}
```

Comment:
It seems that it relies on the host’s environment variables.
I think it would be better to use a default value instead. Users should be able to run CodePair with minimal configuration.